task/01-strings-tasks.js

```diff
@@ -267,7 +274,47 @@ function isString(value) {
  *   'K♠' => 51
  */
 function getCardId(value) {
-    throw new Error('Not implemented');
+    let cardSuits = {
```

Comment:
Most participants use only one array for the task :) But this solution looks interesting too!